### ISSUE TITLE
[CATA][Dungeon][Stonecore] Corborus issues

### ISSUE THREAD

**Issue by true-ede** [NRG-Wow/NRGWow-Bugtracker#208]
**Reporter(Player/Character) Name**:  true-ede

**Description**:  There are several issues with the Corborus boss as of now in the Heroic Stonecore dungeon:

1. Crystal barrage is done too often. Crystals are barely killed off and the next batch spawns like 5 seconds after it.
2. The Thrashing Charge ability doesn't follow the animation of the rumbling ground, so you can get caught by it even if you think you are safe.
3. Occasionally too many Rock Borers spawn.

**Current behaviour**: See above

**Expected behaviour**: Some things should be toned down and the Charge should be aligned to the rumbling ground animation.

**Steps to reproduce the problem**:

1. Try to do Corborus on Heroic

**Comment by Wilcalaf** [comment]
The Crystal Barrage is currently set up to happen between 8 to 10 seconds after fight starts, after that between 10 to 12 seconds and after he emerges 4.5
If you have any reference that this is not like its supposed to please provide otherwise the claim is not valid.

same for the Rock Borers. 

Ill check the second point, for now thats the only valid issue here.

**Comment by Ovahlord** [comment]
The 10 seconds interval for Crystal Barrage is correct.  The initial Brarrage however needs to be settled at 15 seconds instead of 8 - 10.

**Comment by true-ede** [comment]
@Wilcalaf Yeah, you are right about the Barrage, I looked it up. I think I was a "little bit" frustrated when I posted this after a not very successful Stonecore run. :D Sorry about that, I will check the Rock borers again when I get there.

**Comment by hguy23** [comment]
Crystal barage setted at initial to 15 seconds. Waiting for whole re-test of other things.
